lib/build/index.js

```diff
@@ -24,8 +24,8 @@ exports.createLoaderCode = function(p){
   return buildLoaderConfig(project)
       .then(() => {
         let platform = project.build.targets[0];
-        return createLoaderCode(platform, bundler); 
```

Comment:
Just for the sake of viewing "changes", could you remove the changes for the white space here? Nit picky, I know 😛